src/cal.py

```diff
@@ -24,6 +24,24 @@
 from datetime import datetime
 
 
-class calender
+def makecalendar(month=1, year=2019):
+    cal = calendar.TextCalendar(calendar.SUNDAY)
+    formatted = cal.formatmonth(year, month)
+    return formatted
+
+
+date = input(f'Input a date in form month year:  ').split()
+print(date)
+if date:
+    if len(date) == 1:
+        print(makecalendar(int(date[0])))
+    else:
+        print(makecalendar(int(date[1]), int(date[0])))
+        #  this is gross i'll find a better way later hopefully
+else:
```

Comment:
This is a really neat and programmatic way to handle the current date. Good job!